recipes/ruby.rb

```diff
@@ -82,9 +80,22 @@
     rails_env = fetch(:rails_env, "production")
     run "cd #{current_release}; #{rake} RAILS_ENV=#{rails_env} db:seed"
   end
+
+  # This is a simplified port of the deprecated bundler/capistrano task in
+  # https://github.com/rubygems/rubygems/blob/master/bundler/lib/bundler/capistrano.rb
```

Comment:
Might be worth changing this to the permanent link? (Not changed since Aug 2019 but I guess could be deleted someday).

```suggestion
  # https://github.com/rubygems/rubygems/blob/2af2520b4a7ab1c6eb1fdc3d2ef4d8c062d96ad7/bundler/lib/bundler/capistrano.rb
```

Reply:
Good shout. It makes it consistent with the link below. I can't remember if I applied any logic to using a flexible link or not, I was probably lazy.